deployer/src/main.rs

```diff
@@ -1,13 +1,19 @@
 use clap::Parser;
 use shuttle_common::backends::tracing::setup_tracing;
 use shuttle_deployer::args::Args;
+use shuttle_deployer::handlers::make_router;
 use tracing::trace;
 
-#[tokio::main]
+#[tokio::main(flavor = "multi_thread")]
```

Comment:
There is also the general requests processing aspect. Thought that it might be useful for this. What do you think? Of course, I think we can change it whenever even if we go single-threaded for now.

Reply:
It should be multi-threaded by default: https://docs.rs/tokio/latest/tokio/attr.main.html#using-the-multi-thread-runtime , so I'm not sure what this did in the old deployer either :see_no_evil: Perhaps we just needed to enable the multi-threaded feature. Do you remember @chesedo? :smile:

Reply:
Ah nice. Indeed, the multi-threaded feature must be enabled to take effect. I'll change it, thanks!

Reply:
Done.